### PR TITLE
docs: typo fix

### PR DESCRIPTION
**Pull request by kirankumarsripati**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by sarunint** [comment]
@krnsripati Please merge this PR with #26245. There is no need to open multiple pull requests.

**Comment by krnsripati** [comment]
> @krnsripati Please merge this PR with #26245. There is no need to open multiple pull requests.

@sarunint merged typo fixes

File: aio/content/guide/glossary.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ Angular extends TypeScript with domain-specific languages for a number of domain
 
 ## dynamic component loading
 
-A technique for adding a component to the DOM at run time. Requires that you exclude the component from compilation and then connect it to Angular's change-detection and event-handling framework when you add it to the DOM.
+A technique for adding a component to the DOM at runtime. Requires that you exclude the component from compilation and then connect it to Angular's change-detection and event-handling framework when you add it to the DOM.
```
Comment (jenniferfell) [inline aio/content/guide/glossary.md]: It's a fine point of grammar, but "run time" as a noun typically is two words. Please revert this part of your change. Thanks! "runtime" is one word when used as an adjective, such as "runtime environment."

File: aio/content/guide/glossary.md [hunk 2]
```diff
@@ -277,7 +277,7 @@ See also [custom element](guide/glossary#custom-element), which provides an easi
 ## eager loading
 
 NgModules or components that are loaded on launch are called eager-loaded, to distinguish them from those
-that are loaded at run time (lazy-loaded).
+that are loaded at runtime (lazy-loaded).
```
Comment (jenniferfell) [inline aio/content/guide/glossary.md]: at run time

**Review by jenniferfell — CHANGES_REQUESTED** [review]
Thank you for finding these and submitting this PR. One minor request: please revert the changes to "runtime." As a noun, two words is correct. Otherwise, it all looks good. Thanks again!

**Comment by jenniferfell** [comment]
Also, when you're done, please squash the commits. Thanks! 
https://github.com/ExpertSupport/doc-workflow/wiki/Squashing-Commits

**Comment by krnsripati** [comment]
> Also, when you're done, please squash the commits. Thanks!
> https://github.com/ExpertSupport/doc-workflow/wiki/Squashing-Commits

@jenniferfell squashed the commits.

**Review by jenniferfell — APPROVED** [review]

**Pull request closed without merging**